docs/docs/data-stores/README.md

```diff
@@ -158,3 +158,70 @@ But you cannot serialize functions, classes, sets, maps, or other complex object
 You can retrieve up to {{$site.themeConfig.DATA_STORES_MAX_KEYS}} keys from a data store in a single query.
 
 If you're using a pre-built action or code to retrieve all records or keys, and your data store contains more than {{$site.themeConfig.DATA_STORES_MAX_KEYS}} records, you'll receive a 426 error.
+
+## Exporting data to an external service
+
+In order to stay within the [data store limits](#data-store-limits), you may need to export the data in your data store to an external service.
+
+The following Node.js example action will export one batch of data via an HTTP POST request. You may need to adapt the code to your needs.
```

Comment:
Perhaps adding a share workflow link that exposes this step as an HTTP endpoint?

Or a workflow that saves this data to a MySQL/Mongo/Supabase table?

Reply:
Good idea!